src/main/java/edu/whimc/positiontracker/sql/Queryer.java

```diff
@@ -76,11 +105,16 @@ public Queryer(Tracker plugin, Consumer<Boolean> callback) {
         });
     }
 
+    /**
+     * Stores the position data.
+     */
     public void storePositionData() {
+        // get position data for all online players
         final List<PositionEntry> entries = Bukkit.getOnlinePlayers().stream()
                 .map(PositionEntry::new)
                 .collect(Collectors.toList());
 
+        // add data to database if connection exists
```

Comment:
```suggestion
        // add data to database asynchronously
```
This is a [try-with statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) which automatically closes the connection after it's done. Technically the connection doesn't exist at this point.